src/PointwiseFunctions/Hydro/Units.hpp

```diff
@@ -36,6 +36,11 @@ constexpr double rest_mass_density_unit =
 /// `pressure_cgs = pressure_geometric * pressure_unit`
 constexpr double pressure_unit =
     mass_unit / (length_unit * time_unit * time_unit);
+/// CGS Gaussian unit of magnetic field.
```

Comment:
Please add something like `pressure_cgs = pressure_geometric * pressure_unit` to the documentation so it's clear what should be done.